### PR TITLE
Save test updates

### PR DESCRIPTION
**Pull request by inclinedmind**

### CODE REVIEW

**Comment by inclinedmind** [comment]
test comment

File: apps/webapp/tests/e2e/specs/utils/index.ts [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  } else {
+    cy.contains('Connect Wallet').should('be.visible');
+    cy.contains('Connect Wallet').click();
+  }
+
+  cy.contains('MetaMask').should('be.visible');
+  cy.contains('MetaMask').click();
+
+  //skip if we've already connected this address
+  if (skipMetamaskAccess) {
```
Comment (codewiththeflowxyz) [inline apps/webapp/tests/e2e/specs/utils/index.ts:20]: Why not?

```
if (!skipMetamaskAccess) {
    cy.acceptMetamaskAccess().then(connected => {
      expect(connected).to.be.true;
    });
  }
```

**Review by codewiththeflowxyz — APPROVED** [review]
Wow, the tests are now much cleaner. Great job. 

I left a minor comment.

File: apps/webapp/tests/e2e/specs/utils/index.ts (hunk 1 above)
Comment (inclinedmind) [reply apps/webapp/tests/e2e/specs/utils/index.ts:20]: If the address has already been connected to the app, then MM doesn't ask the user for access again

**Review by generatorpoint — APPROVED** [review]
nice, love the reusable functions

**Pull request merged**